Podfile

```diff
@@ -36,3 +39,10 @@ target "LocalNotificationSample" do
     project 'LocalNotificationSample/LocalNotificationSample.xcodeproj'
     foresee_pods
 end
+
+target "FeedbackSample" do
+    project 'FeedbackSample/FeedbackSample.xcodeproj'
+    foresee_pods
+    # include the optional Feedback pod
+    pod 'ForeSee-test/ForeSeeFeedback', '5.2.0-RC19'
```

Comment:
Perhaps we should create a ticket to make sure ForeSee-test will not be used in the master branch?